config/config.hocon.sample

```diff
@@ -60,9 +60,26 @@
       "method": "POST"
     },
 
+    # Sentry-powered crash-reporting; Optional
+    "sentry": {
+      "dsn": "https://sentry.acme.com/42"
+    }
+
     "metrics": {
       # Enable KCL CloudWatch metrics
       "cloudWatch": false
+
+      # StatsD-powered metrics; Optional
+      "statsd": {
+        "hostname": "statsd.acme.ru",
+        "port": 1024,
+        "tags": {
+          # It can resolve environment variables
+          # "worker": ${HOST}
+        }
+        # Optional
+        "prefix": "snowplow.monitoring"
+      }
```

Comment:
Should be in https://github.com/snowplow/snowplow-s3-loader/pull/217/commits/b56ec354a084bf1fd0cd975418f381af17322893